src/app/page.tsx

```diff
@@ -1,95 +1,130 @@
-import Image from 'next/image'
-import styles from './page.module.css'
+import styles from "./page.module.scss";
+
+import {
+  ModeToggleButton,
+  Logo,
+  Searchbar,
+  KeywordRadioBtn,
+} from "./components";
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <p>
-          Get started by editing&nbsp;
-          <code className={styles.code}>src/app/page.tsx</code>
-        </p>
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{' '}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
+    <body>
+      <section>
+        <ModeToggleButton />
+      </section>
+      <header>
+        <Logo />
+      </header>
+      <main>
+        <form name="search">
+          <Searchbar />
+          <fieldset>
+            <KeywordRadioBtn id="name" value="name" />
+            <label htmlFor="name">name</label>
+            <KeywordRadioBtn id="city" value="city" />
+            <label htmlFor="city">city</label>
+            <KeywordRadioBtn id="country" value="country" />
+            <label htmlFor="country">country</label>
+            <KeywordRadioBtn id="any" value="any" />
+            <label htmlFor="any">any keyword</label>
+          </fieldset>
+        </form>
+        <div className={styles.resultCards}></div>
```

Comment:
It will shortly. I left it in because I'm refactoring from an existing project which already contains this class and styles linked  to it. It serves to remind me which classes, elements and styles (will) go together.

Reply:
deleted